package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@lumen5/framefusion",
-  "version": "0.0.16",
+  "version": "0.0.17",
```

```suggestion
  "version": "0.0.18",
```

I just want to force a rebuild